charts/customer-center/Chart.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: customer-center
+description: Chart for Customer-Center application
+type: application
+version: 0.1.0
+appVersion: 2.0.0-rc.1
+keywords:
+  - customer-center
+home: https://my.example.com
+sources:
+  - https://github.com/adfinis-sygroup/customer-center
+  - https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/customer-center
```

I suggest you just link the things repo as it's "home".
```suggestion
home: https://github.com/adfinis-sygroup/customer-center
sources:
  - https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/customer-center
```